apps/vanilla/src/pages/blackjack/Classes/Debugger.ts

```diff
@@ -0,0 +1,18 @@
+import { Subscriber } from '../Types/subscriber';
+
+import { PlayerTurnResult } from '../Types/player-turn-result';
+
+import { Inspector } from './inspector';
+import { ResultData } from './result-displayer';
+
+/** ToDo .*/
+export class Debugger extends Inspector implements Subscriber<PlayerTurnResult> {
+
+	/** Debugger update the dice result base on the DiceGenerator.
+	 * @param playerDiceResult PlayerDuceResult is the dice result of current player.
+	 */
```

Comment:
Please format your multiline jsdoc properly. `/**` and `*/` shouldn't have text for multiline doc.

```suggestion
	/** 
	 * Debugger update the dice result base on the DiceGenerator.
	 * @param playerDiceResult PlayerDuceResult is the dice result of current player.
	 */
```


Also, you can omit subject in the sentence, and be less specific about inner implementation of the function.

```suggestion
	/** 
	 * Update the dice result.
	 * @param playerDiceResult Result of current player.
	 */
```